config/jobs/kubernetes/sig-k8s-infra/trusted/sig-release-release-team-jobs/release-team-periodics.yaml

```diff
@@ -12,6 +12,7 @@ periodics:
     repo: sig-release
     base_ref: master
   spec:
+    suspend: true
```

Comment:
Well, actually this isn't right.  `spec` is a pod spec.  

It doesn't seem like there's a straightforward way to pause a periodic unless we wanted to break something about the job so it stops working (changing the command, or project number). 

Or we could potentially accomplish the same thing by restricting the use of the lead-opted-in label (set it to an empty list) - https://github.com/kubernetes/test-infra/blob/master/config/prow/plugins.yaml#L175-L176